action/protocol/execution/evm/contract_test.go

```diff
@@ -32,8 +33,10 @@ func TestCreateContract(t *testing.T) {
 	require := require.New(t)
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
-	testTrieFile, _ := ioutil.TempFile(os.TempDir(), "trie")
+	testTrieFile, err := ioutil.TempFile(os.TempDir(), "trie")
+	require.NoError(err)
 	testTriePath := testTrieFile.Name()
+	require.NoError(testTrieFile.Close())
```

Comment:
so many duplicate code that it worths the effort to add a function into testutil
```
func PathOfTempFile(dirname string) (string, error) {
    tempFile, err := ioutil.TempFile(os.TempDir(), dirname)
    if err != nil {
        return "", err
    }
    return tempFile.Name(), tempFile.Close()
}

Reply:
ok